### PR TITLE
Added a shortcut to return a pager for hybrid results

### PR DESCRIPTION
**Pull request by COil**
This is a method I have added in one of my project. It would be nice to have it in the bundle. 
Sorry, I made this quite rapidly. What do I need to add to the PR?

### CODE REVIEW

**Comment by COil** [comment]
Hi @XWB , should the method be added in the interface?

**Comment by COil** [comment]
> Hi @XWB , should the method be added in the interface?

I have added the method in the interface.

**Comment by COil** [comment]
You can see how it can be used here: https://www.strangebuzz.com/en/blog/implementing-a-search-engine-with-elasticsearch-and-symfony-part-2?q=elasticsearch#h2_5

**Comment by COil** [comment]
Hi @lsmith77 , can it be merged please?

File: src/Finder/TransformedFinder.php [hunk 1]
```diff
@@ -80,6 +80,16 @@ public function findPaginated($query, $options = [])
         return new Pagerfanta(new FantaPaginatorAdapter($paginatorAdapter));
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function findHybridPaginated($query)
+    {
+        $paginatorAdapter = $this->createHybridPaginatorAdapter($query);
```
Comment (lsmith77) [inline src/Finder/TransformedFinder.php]: don’t we need some code here to ensure we have a query instance?

Comment (COil) [reply src/Finder/TransformedFinder.php]: Hi @lsmith77  , it's done in `createHybridPaginatorAdapter` -> `$query = Query::create($query);` -> `throw new InvalidException('Unexpected argument to create a query for.');`

Comment (lsmith77) [reply src/Finder/TransformedFinder.php]: ah thx.

Comment (COil) [reply src/Finder/TransformedFinder.php]: > I guess this is convenient syntax sugar?

Yes. It avoid me to add this function at two places in a project : 
https://www.strangebuzz.com/en/blog/implementing-a-search-engine-with-elasticsearch-and-symfony-part-2 
Check out the commented line.

> shouldn’t we then also one such method using createRawPaginatorAdapter()?

I am not sure to understand what you mean.

Comment (lsmith77) [reply src/Finder/TransformedFinder.php]: ie. adding a `findRawPaginated()`

Comment (COil) [reply src/Finder/TransformedFinder.php]: > ah thx.

The `$queryObject = Query::create($query);` call in the `findPaginated` function seems useless as it's already done in `createPaginatorAdapter` function.

**Pull request merged**